scopes/react/react/docs/docs-app.tsx

```diff
@@ -5,6 +5,8 @@ import { ThemeContext } from '@teambit/documenter.theme.theme-context';
 import { IconFont } from '@teambit/design.theme.icons-font';
 import { RenderingContext } from '@teambit/preview';
 import { Base } from './base';
+// @TODO - delete this once using @teambit/documenter.theme.theme-context > v40.0.0
```

Comment:
@KutnerUri I guess it should be >4.0.0 not 40.0.0?

Reply:
fixed.